h/services/annotation_write.py

```diff
@@ -82,7 +82,7 @@ def create_annotation(self, data: dict) -> Annotation:
             self._annotation_metadata_service.set(annotation, annotation_metadata)
 
         self._queue_service.add_by_id(
-            name=JobQueueService.JobName.SYNC_ANNOTATION,
```

Comment:
Alternatively we could do `.value` here but I'd like to see if we can avoid that in a separate PR.